scripts/database.js

```diff
@@ -249,3 +249,15 @@ const database = {
     },
   ],
 };
+
+export const getSchools = () => {
+  return database.schools.map(schools => ({ ...schools }))
```

Comment:
The iterator should be singular, not plural, as it represents one school object in the schools array.